cookiecutter.json

```diff
@@ -27,5 +27,6 @@
     "Vue3",
     "React",
     "None"
-  ]
+  ],
+  "include_mobile": "n"
```

Comment:
I think this could be a clearer question?
So all this will do will add in react native login? Does this require picking react above to work? If they want a mobile app should they even be allowed to pick web as well? or only have react as an option?

Reply:
Weird question 🤔  why would that not be allowed?
I like the idea of having the client as an option, for Julep for example we are likely going to need Webviews to work.

Reply:
We could do `include_mobile_project` ?
This is an independent question from web clients, you can choose whatever client or even none and still get a react native project if you answer this as `yes`

Reply:
OK so you could pick Vue and mobile and everything should be fine (though I imagine most wouldn't do that)